lib/puppet/pops/evaluator/compare_operator.rb

```diff
@@ -88,27 +88,27 @@ def cmp_Symbol(a, b)
     if b.is_a?(Symbol)
       a <=> b
     else
-      raise ArgumentError.new("Symbol not comparable to non Symbol")
+      raise ArgumentError.new(_("Symbol not comparable to non Symbol"))
     end
   end
 
   def cmp_Timespan(a, b)
-    raise ArgumentError.new('Timespans are only comparable to Timespans, Integers, and Floats') unless b.is_a?(Time::Timespan) ||  b.is_a?(Integer) || b.is_a?(Float)
+    raise ArgumentError.new(_('Timespans are only comparable to Timespans, Integers, and Floats')) unless b.is_a?(Time::Timespan) ||  b.is_a?(Integer) || b.is_a?(Float)
     a <=> b
   end
 
   def cmp_Timestamp(a, b)
-    raise ArgumentError.new('Timestamps are only comparable to Timestamps, Integers, and Floats') unless b.is_a?(Time::Timestamp) ||  b.is_a?(Integer) || b.is_a?(Float)
+    raise ArgumentError.new(_('Timestamps are only comparable to Timestamps, Integers, and Floats')) unless b.is_a?(Time::Timestamp) ||  b.is_a?(Integer) || b.is_a?(Float)
     a <=> b
   end
 
   def cmp_Version(a, b)
-    raise ArgumentError.new('Versions not comparable to non Versions') unless b.is_a?(SemanticPuppet::Version)
+    raise ArgumentError.new(_('Versions not comparable to non Versions')) unless b.is_a?(SemanticPuppet::Version)
     a <=> b
   end
 
   def cmp_Object(a, b)
-    raise ArgumentError.new('Only Strings, Numbers, Timespans, Timestamps, and Versions are comparable')
+    raise ArgumentError.new(_('Only Strings, Numbers, Timespans, Timestamps, and Versions are comparable'))
```

Comment:
We can add these terms to our glossary to ensure that they are translated consistently.